file-formats/uipg/examples/(ui2)flp_demo_page.uipg.json

```diff
@@ -0,0 +1,238 @@
+{
+  "formatVersion": "1",
+  "header": {
+    "description": "UI2 FLP Demo - Test Page",
+    "originalLanguage": "en"
+  },
+  "generalInformation": {
+    "title": "UI2 FLP Demo - Test Page",
+    "description": "This page is used for testing the pages runtime"
+  },
+  "sections": [
+    {
+      "id": "id-1566210576050-45",
```

Comment:
This is kind of hard to change.

The tileAssignment ones are referencing other entities so here we have no choice.

Others may in theory be changed, but we do have to ensure that they are kept stable because there are other objects referencing these. (e.g. personalization in the FLP or WZ).

At least endusers don't interact with these IDs in a visible manner.